src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -178,7 +178,8 @@ def on_time_step(self, event: Event):
             working_age_idx, self.randomness, self.job_change_rate, "changing jobs"
         )
         moved_idx = pop.index[pop["address_id"] != pop["previous_timestep_address_id"]]
-        changing_jobs_idx = changing_jobs_idx.union(moved_idx.intersection(working_age_idx))
+        moved_working_age_idx = moved_idx.intersection(working_age_idx)
+        changing_jobs_idx = changing_jobs_idx.union(moved_working_age_idx)
```

Comment:
We have a test that people under 18 remain unemployed. I'm kind of surprised that didn't fail prior to this fix. Can you see why that was? It's a sign that our test coverage had a gap.

Reply:
This diff didn't change any logic -- just moved `moved_idx.intersection(working_age_idx)` onto its own line/variable.